Serilog.Sinks.Network/NetworkLoggerConfigurationExtensions.cs

```diff
@@ -41,42 +41,33 @@ public static LoggerConfiguration TCPSink(
             this LoggerSinkConfiguration loggerConfiguration,
             IPAddress ipAddress,
             int port,
-            int? writeTimeoutMs = null,
-            int? disposeTimeoutMs = null,
             ITextFormatter? textFormatter = null,
-            LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum)
-        {
-            return TCPSink(loggerConfiguration, $"tcp://{ipAddress}:{port}", writeTimeoutMs, disposeTimeoutMs,  textFormatter, restrictedToMinimumLevel);
-        }
-
-        public static LoggerConfiguration TCPSink(
-            this LoggerSinkConfiguration loggerConfiguration,
-            IPAddress ipAddress,
-            int port,
-            LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum)

```

Comment:
This overload
```csharp
        public static LoggerConfiguration TCPSink(
            this LoggerSinkConfiguration loggerConfiguration,
            IPAddress ipAddress,
            int port,
            LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum)
```
was added in #51 included in 3.0.0-alpha.1 but I think can be safely removed. It can cause ambiguous call errors with code like `TCPSink(IPAddress.Parse("127.0.0.1"), 1234)` because the overload above could also match.

I think this is OK to remove because the overload above can cover the same functionality/parameters with calls like this:
`TCPSink(IPAddress.Parse("127.0.0.1"), 1234, restrictedToMinimumLevel: LogEventLevel.Debug)`